src/Jet.JsonNet.Converters/UnionConverter.fs

```diff
@@ -61,7 +61,7 @@ module private Union =
 type UnionConverter private (discriminator : string, ?catchAllCase) =
     inherit JsonConverter()
 
-    new(discriminator: string, catchAllCase: string) = UnionConverter(discriminator, ?catchAllCase=Option.ofObj catchAllCase)
+    new(discriminator: string, catchAllCase: string) = UnionConverter(discriminator, ?catchAllCase = match catchAllCase with null -> None | x -> Some x)
```

Comment:
defaultArg ;)

Reply:
Brain fade - your code is correct of course!

Reply:
though I've always liked code like: `if catchAllCase = null then None else Some catchAllCase` ;) (and it will make CI run!)